src/extension/content-script/onstart.ts

```diff
@@ -12,9 +12,6 @@ async function onstart() {
   }
 
   const account = await api.getAccount();
-  // window.webln
-  injectScript(browser.runtime.getURL("js/inpageScriptWebLN.bundle.js"));
```

Comment:
@pavanjoshi914 can you inject all the providers below in this way? I want to test it with `window.liquid` for example.

Reply:
@pavanjoshi914 this needs to be addressed still, right?